src/SME.SGP.Aplicacao/Consultas/ConsultasNotasConceitos.cs

```diff
@@ -84,6 +91,13 @@ public async Task<NotasConceitosRetornoDto> ListarNotasConceitos(ListaNotasConce
                     var notas = repositorioNotasConceitos.ObterNotasPorAlunosAtividadesAvaliativas(atividadesAvaliativasdoBimestre.Select(a => a.Id).Distinct(), alunosIds, filtro.DisciplinaCodigo);
                     var ausenciasAtividadesAvaliativas = await repositorioFrequencia.ObterAusencias(filtro.TurmaCodigo, filtro.DisciplinaCodigo, atividadesAvaliativasdoBimestre.Select(a => a.DataAvaliacao).Distinct().ToArray(), alunosIds.ToArray());
 
+                    var disciplinaEOL = servicoEOL.ObterDisciplinasPorIds(new long[] { long.Parse(filtro.DisciplinaCodigo) }).FirstOrDefault();
+                    IEnumerable<DisciplinaResposta> disciplinasRegencia = null;
+
+                    if (disciplinaEOL.Regencia)
```

Comment:
Se a disciplina não for encontrada (for null) vai dar erro.

Reply:
essa disciplina que vem no filtro é informada na tela